src/main/java/com/google/sps/data/Accessory.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+public class Accessory {
+
+  public enum Type {
+    HAT,
+    GLASSES,
+    COMPANION
+  };
+
+  private String id;
+  private String imageFilePath;
+  private Type type;
+  private int height;
+  private int width;
+  private int xPos;
```

Comment:
How are x/y positions defined? Relative to the image? Does it matter if it is on web vs mobile?

Reply:
add comments about it

Reply:
Relevant to the image container that holds the sprite avatar and body, I have been testing it out on the front end. I would also like to add an admin only page that allows you to add accessories by dragging and dropping onto the sprite.